lib/decoders/shapefile.js

```diff
@@ -57,12 +58,14 @@ class Shapefile extends Duplex {
     return `/tmp/${name}_${this._fgroup}${extension}`;
   }
 
-  write(chunk, encoding, done) {
-    logger.debug(`Writing shape feature with ${encoding} encoding`);
-    return this._zBuffer.write(chunk, encoding, done);
+  _write(chunk, encoding, done) {
+    logger.debug(`Writing shape feature chunk ${typeof chunk} with ${encoding} encoding`);
+    return this._zBuffer.write(new Buffer(chunk.toString('binary'), 'binary'), 'binary', done);
```

Comment:
i've actually removed it - was messing around trying to fix encoding issues and someone on the internet said it was required, but it's not - encoding is coming in as binary so there are other issues lurking.